### PR TITLE
Adding Feature of Create Course by Connecting Create Course Form at Backend

### PR DESCRIPTION
**Pull request by JavidSumra**
- [x] *Have you read the [Contributing Guidelines](https://github.com/jessesquires/.github/blob/master/CONTRIBUTING.md)?*

Issue #98 

## Describe your changes
Adding `handleSubmit` Method To Handle the Submission of Form And Then Creating A Request to Backend.

image

image

### CODE REVIEW

**Comment by JavidSumra** [comment]
Hey @anand-harsh Sir Please Checkout My PR.

File: client/src/components/Admin/CreateCourse/CreateCourse.jsx [hunk 1]
```diff
@@ -37,6 +42,52 @@ const CreateCourse = () => {
...
+      const res = await fetch(`${API_ENDPOINT}/createcourse`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ title, description, createdBy, category }),
+        credentials: 'include',
+      });
+
+      const data = await res.json();
```
Comment (anand-harsh) [inline client/src/components/Admin/CreateCourse/CreateCourse.jsx:57]: Remove console.log statement

**Review by anand-harsh — CHANGES_REQUESTED** [review]

**Comment by JavidSumra** [comment]
Changes Done @anand-harsh Sir

**Review by anand-harsh — APPROVED** [review]
LGTM✨✨

**Pull request merged**